src/gui/FileBrowser.cpp

```diff
@@ -403,11 +340,11 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 						this,
 					SLOT( sendToActiveInstrumentTrack() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"Song-Editor" ),
+								"Song Editor" ),
 						this,
 					SLOT( openInNewInstrumentTrackSE() ) );
 		contextMenu.addAction( tr( "Open in new instrument-track/"
-								"B+B Editor" ),
+								"Pattern Editor" ),
```

Comment:
@tresf I don't see the inconsistency. Can you clarify what the issue is?
On Sep 28, 2015 8:54 AM, "Tres Finocchiaro" <notifications@github.com>
wrote:

> In src/gui/FileBrowser.cpp
> <https://github.com/LMMS/lmms/pull/2331#discussion_r40568894>:
>
> >  						this,
> >  					SLOT( openInNewInstrumentTrackSE() ) );
> >  		contextMenu.addAction( tr( "Open in new instrument-track/"
> > -								"B+B Editor" ),
> > +								"Pattern Editor" ),
>
> @Wallacoloo <https://github.com/Wallacoloo> I'm not sure this got caught,
> but this cause inconsistencies in the interface...
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/6345473/10140878/af8dfbca-65d7-11e5-9ef5-6888862cfe86.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LMMS/lmms/pull/2331/files#r40568894>.
>